juntagrico/templates/exports/share_certificate.html

```diff
@@ -0,0 +1,138 @@
+{% load i18n %}
+{% load config %}
+<!DOCTYPE html>
+<html lang="de">
+<head>
+<title>Bescheinigung</title>
+<meta charset="utf-8">
+<style>
+    body {
+        font-family: Helvetica;
+        font-size: 12pt;
+    }
+
+     @page {
+        size: A4;
+        @frame content_frame {
+            left: 50pt;
+            width: 512pt;
+            top: 45pt;
+            height: 710pt;
+    /*        -pdf-frame-border: 1;    */    /* for debugging the layout */
+        }
+
+        @frame header_frame {
+            -pdf-frame-content:header_content;
+            left: 50pt;
+            width: 512pt;
+            top: 50pt;
+            height: 20pt;
+            text-align:center;
+        }
+
+        @frame footer_frame {
+            -pdf-frame-content:footer_content;
+            left: 50pt;
+            width: 512pt;
+            top: 760pt;
+            height: 90pt;
+            text-align:center;
+        }
+    }
+
+    .org_address {
+        text-align: right;
+     }
+
+    table {
+        width: 200px;
+        text-align: left;
+        margin-left: auto;
+        margin-right: auto;
+    }
+
+    td, th {
+        text-align: left;
+    }
+
+    .member {
+        display: inline-block;
+        margin-left: 1em;
+    }
+
+    .letter_date {
+        padding-top: 60px;
+    }
+
+    .subject {
+        padding-top: 20px;
+        padding-bottom: 20px;
+    }
+</style>
+</head>
+<body>
+{{ debugdate }}
+{% block content %}
+    {% vocabulary "share" as v_share %}
+    {% vocabulary "share_pl" as v_share_pl %}
+    {% vocabulary "member" as v_member %}
+    {% vocabulary "subscription" as v_subscription %}
+    {% config "organisation_address" as organisation_address %}
+    {% config "currency" as currency %}
+    {% config "share_price" as share_price %}
```

Comment:
wenn wir schon dabei sind sollten wir dem share ein value feld geben damit sich der share price über die jahre ändern kann

Reply:
Das erhöht die Komplexität ordentlich. Hinzu kämen die Migration, das Certificate müsste unterschiedliche Zeilen für unterschiedliche Share-Werte im Jahr auflisten und eine Gesamtsumme hinzukommen. Korrekturen vom Sharewert werden kompliziert.

In diesen Pullrequest einbauen oder später upgraden?

Reply:
hier gleich einbauen, weil wenn wir schon ein ofizielles dokument generieren müssen wir auch garantieren können, dass wir keinen unsinn rausgeben und im moment wäre das möglich wenn jemand unbedarft am entsprechenden setting rumschraubt. das wäre so oder so ein problem aber eines das intern bleiben würde und mit dem zertifikat wäre es ein problem das nach aussen getragen wird. dh das in einem anteilschein drinsteht wieviel er wert ist ist eigentlich die grundvoraussetzung das wir dan das zertifikat anbieten können

Reply:
Done